README.md

```diff
@@ -1,8 +1,58 @@
-# Open Container - Container Distributable Image Specification
+# Open Container - Distributable Image Specification
 
-[Open Container Initiative](http://www.opencontainers.org/) Specifications for standards on Operating System process and application containers.
+The [Open Container Initiative](http://www.opencontainers.org/) develops specifications for standards on Operating System process and application containers.
 
-https://github.com/opencontainers/tob/tree/master/proposals/image-format
+This OCI project is tasked with creating a software shipping container image format spec (OCI Image Format) with security and naming as components.
+
+## Initial Proposal
+
+This new OCI project intends to start with the Docker v2.2 specification, improve any remaining technical concerns, and standardize and improve the understood properties of a container image format. This new project will have the objectives of:
+
+* A serialized image format (base layer)
+* A process of hashing the image format for integrity and content-addressing (base layer)
+* Signatures that are based on signing image content address (optional layer)
+* Naming that is federated based on DNS and can be delegated (optional layer)
+
+## Cooperation with OCI Runtime Project
+
+The [OCI Runtime Spec project](https://github.com/opencontainers/runtime-spec) is working diligently to create a specification for the lifecycle of a running container. The OCI Image Format Spec project should work with the OCI Runtime Spec project so that the image can support the UX that users have come to expect from container engines like Docker and rkt: primarily, the ability to run an image with no additional arguments:
```

Comment:
fixed

On Mon, Apr 4, 2016 at 10:01 PM Brendan Burns <notifications@github.com>
wrote:

> In README.md
> <https://github.com/opencontainers/image-spec/pull/5#discussion_r58438091>
> :
>
> >
> > -https://github.com/opencontainers/tob/tree/master/proposals/image-format
> > +This OCI project is tasked with creating a software shipping container image format spec (OCI Image Format) with security and naming as components.
> > +
> > +## Initial Proposal
> > +
> > +This new OCI project intends to start with the Docker v2.2 specification, improve any remaining technical concerns, and standardize and improve the understood properties of a container image format. This new project will have the objectives of:
> > +
> > +* A serialized image format (base layer)
> > +* A process of hashing the image format for integrity and content-addressing (base layer)
> > +* Signatures that are based on signing image content address (optional layer)
> > +* Naming that is federated based on DNS and can be delegated (optional layer)
> > +
> > +## Cooperation with OCI Runtime Project
> > +
> > +The [OCI Runtime Spec project](https://github.com/opencontainers/runtime-spec) is working diligently to create a specification for the lifecycle of a running container. The OCI Image Format Spec project should work with the OCI Runtime Spec project so that the image can support the UX that users have come to expect from container engines like Docker and rkt: primarily, the ability to run an image with no additional arguments:
>
> rephrase?
>
> "The OCI Runtime Spec project is developing a specification for the
> lifecycle..."
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/5/files/98006c3f6ac80d253f67570cf52bbd0f19a1f6b8#r58438091>
>